src/main/java/uk/gov/hmcts/cmc/claimstore/documents/DocumentUploader.java

```diff
@@ -3,50 +3,42 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.context.event.EventListener;
 import org.springframework.stereotype.Component;
-import uk.gov.hmcts.cmc.claimstore.documents.output.PDF;
 import uk.gov.hmcts.cmc.claimstore.events.DocumentGeneratedEvent;
 import uk.gov.hmcts.cmc.claimstore.services.ClaimService;
-import uk.gov.hmcts.cmc.claimstore.services.document.DocumentManagementService;
-
-import java.net.URI;
-
-import static uk.gov.hmcts.cmc.claimstore.utils.DocumentNameUtils.isSealedClaim;
+import uk.gov.hmcts.cmc.claimstore.services.document.DocumentsService;
 
 @Component
 @ConditionalOnProperty(prefix = "document_management", name = "url")
 public class DocumentUploader {
     private static final Logger logger = LoggerFactory.getLogger(DocumentUploader.class);
-    private final DocumentManagementService documentManagementService;
+    private final DocumentsService documentService;
     private final ClaimService claimService;
 
     @Autowired
-    public DocumentUploader(DocumentManagementService documentManagementService,
+    public DocumentUploader(@Qualifier("DocumentManagementBackedDocumentsService") DocumentsService documentService,
```

Comment:
do we need this qualifier as well as does DocumentManagementBackedDocumentsService need alias

Reply:
yes , because there are two implementations of DocumentService.

Reply:
alias can be same for both implementation , then we don't need this qualifier